python/src/main/python/drivers/graphicsfuzz_piglit_converter.py

```diff
@@ -182,8 +182,6 @@ def get_json_properties(shader_job: str) -> List:
     """
     with gfuzz_common.open_helper(shader_job, 'r') as job:
         json_parsed = json.load(job)
```

Comment:
By "blank JSON", do you mean an empty file, or a file containing "{}"?  When there are no uniforms we tend to provide the latter, and I believe that should be parsed into an empty dictionary, which your original code should handle just fine.  Are you talking about an actual empty file?  If so, what's the use case for that?

Reply:
OK, thanks to @paulthomson I have understood what's going on here - an empty dictionary evaluates to false, so you're to have removed the check.